README.md

```diff
@@ -34,7 +36,6 @@ metadata:
   namespace: hypershift
```

Comment:
Note for later: should probably omit the `hypershift` namespace.

Reply:
Fixed this in the README